pkgs/dart_tooling_mcp_server/test_fixtures/counter_app/lib/main.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
```

Comment:
copright

Reply:
I am not sure how to handle this... this is just the vanilla counter sample app.

I could just not check it in and have the test run `flutter create` to sidestep things?

Reply:
Interesting idea. How much time does it take to run flutter create? If we limited the `flutter create` command to just the platform we intend on using, that could speed it up. We'd just need to make sure that the location it is created in is cleaned up at the end of the test (pass or failure).

Reply:
As discussed since we probably will want to alter the app to do things like report runtime errors, we do need a real checked in app